docs/ops/activation/SoftPlus_4.md

```diff
@@ -0,0 +1,45 @@
+## SoftPlus <a name="SoftPlus"></a>
+
+**Versioned name**: *SoftPlus-4*
+
+**Category**: *Activation*
+
+**Short description**: SoftPlus takes one input tensor and produces output tensor where the softplus function is applied to the tensor elementwise.
+
+**Detailed description**: For each element from the input tensor calculates corresponding
+element in the output tensor with the following formula: `features[i] = log(exp(features[i]) + 1)`
+
+**Attributes**: *SoftPlus* operation has no attributes.
+
+
+**Inputs**:
+
+*   **1**: Multidimensional input tensor of type *T*. **Required**.
+
+**Outputs**:
+
+*   **1**: The resulting tensor of the same shape and type as input tensor.
+
+**Types**
+
+* *T*: any supported type.
```

Comment:
I think that we want to support only floating point types (not integer and boolean).

Reply:
Ok, updated.